server/README.md

```diff
@@ -6,19 +6,40 @@
 - Clone Kommunicate-Web-SDK repository on your machine. ([How to clone a Github repository ?](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository))
 
 
-## Steps to deploy Kommunicate server.
+## How to run Kommunicate-Web-SDK in localhost.
 Step-1: Open Terminal on your machine.
 
 Step-2: Change the current working directory to Kommunicate-Web-SDK directory.
 
-Step-3: Move to the server folder in Kommunicate-Web-SDK directory.
-> $ cd server
+Step-3: Open the terminal and run this command to install the dependencies
+> $ npm install
+
+Step-3: To run the Kommunicate-Web-SDK in local you need to run this command in terminal
+> $ npm run dev
 
-Step-4  To run the node server run the below command.
-> $ NODE_ENV=prod node app
 
 Step-5: Open your web browser and enter http://localhost:3030 to run the demo.
 
+## How to debug the Production Kommunicate-Web-SDK in localhost.
+
+> Note -> These steps will be applicable for all environments like -> test, release, prod_in, prod etc. Make sure update only that environment key and hostUrl.
```

Comment:
"Make sure update only that environment key and hostUrl." 
->
Please ensure that you update only the environment key and hostUrl

Reply:
ok